### ISSUE TITLE
wrong "unconfirmed" label for transactions that leave 0 change

### ISSUE THREAD

**Issue by aleqx** [mimblewimble/grin-wallet#169]
Confirmed this on 1.1.0 and looking at the 2.0.0 changes the v2.0.0 will have the same problem.

If I send all GRIN in the wallet, thus leaving 0 change, the wallet will forever display the transaction as unconfirmed, even though it has been mined and included in the chain with plenty of confirmations.

Discussing this on Gitter, @jaspervdm explained that there is a way to check tx status by kernel instead of change output, but that it hasn't been implemented yet.

I hereby put in my plea for that to be implemented _please_ :smile:

**Comment by yeastplume** [comment]
Agreed, this was just too big a change for 2.0.0 with everything else going on, and also needs additions on the node side to support this. Tentatively scheduling this for 2.0.1

**Comment by bladedoyle** [comment]
This has been reported multiple times.
see also: https://github.com/mimblewimble/grin-wallet/issues/69

**Comment by aleqx** [comment]
This can also mess up exchanges that use the official cli wallet to check if a withdrawal was successful ...

**Comment by antiochp** [comment]
Do we have an issue tracking what's needed on the node side of things?

**Comment by yeastplume** [comment]
@antiochp I don't think we do (but need to look through to be sure)
